src/components/hero.jsx

```
@@ -75,61 +75,61 @@
-          className='clear-button'
-          onClick={ () => {
-            setSearchTerm('');
-            setSelectedBreedId(null);
-          }}
+          {searchTerm && <button 
+            className='rounded-full text-2xl text-gray-800 mt-2.5 p-3 border-none bg-white transition-colors duration-300 cursor-pointer hover:bg-gray-300'
+            onClick={ () => {
+              setSearchTerm('');
+              setSelectedBreedId(null);
+            }}
           ><MdClear /></button>}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)